tensorflow/core/kernels/BUILD

```diff
@@ -193,6 +195,16 @@ tf_cc_test(
     ],
 )
 
+# virtual targets since nested select statements not possible
```

Comment:
nice bazel usage here!

Reply:
Thanks!  I can't take full credit.  I got the idea from https://github.com/bazelbuild/bazel/issues/1623#issuecomment-369909088